app/javascript/packs/saml-post.js

```diff
@@ -1,4 +1,4 @@
 document.addEventListener('DOMContentLoaded', function () {
-  document.body.className += ' hide';
+  document.body.className += ' display-none';
```

Comment:
This is for hiding the "JavaScript disabled, click to continue" screen shown after SAML authentication.

Suggesting to revert back to a screen-reader-only text, since...

1. We rely on following line to submit a form within this hidden region, which apparently can be a problem in some legacy browsers for `display: none` ([source](https://stackoverflow.com/questions/8318428/submit-form-fields-inside-displaynone-element))
2. It could be jarring for assistive technology if the content exists in the page at load and then is suddenly removed via JavaScript

```suggestion
  document.body.className += ' usa-sr-only';
```